gnome-screenlock-password-opensuse132-20160620.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "type": "match",
+      "ypos": 237,
+      "height": 263,
+      "width": 458,
+      "xpos": 290
+    }
+  ],
+  "tags": [
+    "gnome-screenlock-password"

```

Comment:
I don't think you need DESKTOP-gnome in a needle that already has gnome in its tag. And the DISTRI-opensuse part is really obsolete as we only have one needle repo for it. But the VERSION-13.2 one will make it possible not to check it on any other version (which I think is not yet implemented in main.pm)